snapcraft/internal/lxd.py

```diff
@@ -120,6 +142,36 @@ def _wait_for_network(self):
         logger.info('Network connection established')
 
 
+class Cleanbuilder(Containerbuild):
+
+    def __init__(self, snap_output, source_folder, project_options,
+                 config, remote=None):
+        container_name = petname.Generate(3, '-')
+        super().__init__(snap_output, source_folder, project_options,
+                         config, container_name, remote)
+
+
+class Project(Containerbuild):
+
+    def __init__(self, snap_output, source_folder, project_options,
+                 config, remote=None):
+        super().__init__(snap_output, source_folder, project_options,
+                         config, config.data['name'], remote)
+
+    def _create_container(self):
```

Comment:
it would be better here to do maybe a `_ensure_container(container_name)` (akin to `_verify_remote`) and then move to `_ensure_started(container_name)`